src/pages/professors.tsx

```diff
@@ -1,30 +1,28 @@
 import {
```

Comment:
make sure to run prettier
or `npm run format`

Reply:
also in C2Table pls
https://www.digitalocean.com/community/tutorials/code-formatting-with-prettier-in-visual-studio-code#:~:text=To%20this!-,Automatically%20Format%20on%20Save,-So%20far%2C%20we